charts/daps-reg-service/Chart.yaml

```diff
@@ -32,10 +32,13 @@ description: A Helm chart for Kubernetes
 # pipeline. Library charts do not define any templates and therefore cannot be deployed.
 type: application
 
+sources:
+  - https://github.com/catenax-ng/tx-daps-registration-service
```

Comment:
This is not a valid source repository for Eclipse-Tractusx. Please use https://github.com/eclipse-tractusx/daps-registration-service instead.